### PR TITLE
Replace old logo

### PR DESCRIPTION
**Pull request by Nightcaat**
Website is untested but should be fine.

Thanks to afn, PickleNik, and everyone who gave feedback :]

### CODE REVIEW

File: Extensions/chrome/popup.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <center>
-      <img src="icons/icon48.png" alt="Logo" />
+      <svg width="48" viewBox="0 0 24 24"><path d="M14.9 3H6c-.9 0-1.6.5-1.9 1.2l-3 7c-.1.3-.1.5-.1.7v2c0 1.1.9 2 2 2h6.3l-.9 4.5c-.1.5 0 1 .4 1.4l1.1 1.1 6.5-6.6c.4-.4.6-.9.6-1.4V5c-.1-1.1-1-2-2.1-2zm7.4 12.8h-2.9c-.4 0-.7-.3-.7-.7V3.9c0-.4.3-.7.7-.7h2.9c.4 0 .7.3.7.7V15c0 .4-.3.8-.7.8z" fill="red"/><path d="m8 12.5 5.1-2.9L8 6.7v5.8z" fill="#fff"/></svg>
```
Comment (PickleNik) [inline Extensions/chrome/popup.html:13]: 🤔 maybe svg?

File: Extensions/firefox/popup.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 
     <center>
-      <img src="icons/icon48.png" alt="Logo" />
+      <svg width="48" viewBox="0 0 24 24"><path d="M14.9 3H6c-.9 0-1.6.5-1.9 1.2l-3 7c-.1.3-.1.5-.1.7v2c0 1.1.9 2 2 2h6.3l-.9 4.5c-.1.5 0 1 .4 1.4l1.1 1.1 6.5-6.6c.4-.4.6-.9.6-1.4V5c-.1-1.1-1-2-2.1-2zm7.4 12.8h-2.9c-.4 0-.7-.3-.7-.7V3.9c0-.4.3-.7.7-.7h2.9c.4 0 .7.3.7.7V15c0 .4-.3.8-.7.8z" fill="red"/><path d="m8 12.5 5.1-2.9L8 6.7v5.8z" fill="#fff"/></svg>
```
Comment (PickleNik) [inline Extensions/firefox/popup.html:15]: 🤔 maybe svg?

File: Website/pages/index.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
 
-    <svg id="thumbslogo" class="mb-4" width="150" height="150" viewBox="0 0 24 24"><path d="m15 3h-9c-0.83 0-1.54 0.5-1.84 1.22l-3.02 7.05c-0.09 0.23-0.14 0.47-0.14 0.73v2a2 2 0 0 0 2 2h6.31l-0.95 4.57a1.49 1.49 0 0 0 0.41 1.37l1.06 1.06 6.58-6.59c0.37-0.36 0.59-0.86 0.59-1.41v-10a2 2 0 0 0-2-2zm4 12h4v-12h-4z" /><path d="m7.47 12.2 4.76-2.7-4.76-2.71z" fill="#fff" stroke="none"/></svg>
+    <svg id="thumbslogo" class="mb-4" width="150" viewBox="0 0 24 24"><path d="M14.9 3H6c-.9 0-1.6.5-1.9 1.2l-3 7c-.1.3-.1.5-.1.7v2c0 1.1.9 2 2 2h6.3l-.9 4.5c-.1.5 0 1 .4 1.4l1.1 1.1 6.5-6.6c.4-.4.6-.9.6-1.4V5c-.1-1.1-1-2-2.1-2zm7.4 12.8h-2.9c-.4 0-.7-.3-.7-.7V3.9c0-.4.3-.7.7-.7h2.9c.4 0 .7.3.7.7V15c0 .4-.3.8-.7.8z"/><path d="m8 12.5 5.1-2.9L8 6.7v5.8z" fill="#fff" stroke="none"/></svg>
```
Comment (PickleNik) [inline Website/pages/index.vue:4]: ⚠️ `Warning`: not specifying the height might cause anxiety issues

**Review by PickleNik — COMMENTED** [review]
👀

**Comment by Nightcaat** [comment]
Not comfortable having this merged yet, looking into how the Discord version of the logo will appear on the latest version of the mobile apps

**Review by PickleNik — APPROVED** [review]

**Comment by PickleNik** [comment]
merging this to avoid conflict with #274

**Pull request merged**